CHANGELOG.md

```diff
@@ -306,7 +305,7 @@ Versioning](https://semver.org/spec/v2.0.0.html).
   - `StorageKey`: A structure that stores lookup keys generated by `StorageKeyBuilder`.
   - `KeyExtractor`: A trait for calling the value to be set by the key.
 - Supports 14 sysmon events.
-- Added `search sysmon event` Graphql Api for `sysmon` supported by giganto.
+- Added `sysmonEvents` Graphql API for `sysmon` supported by giganto.
```

Comment:
This part refers to queries of the search sysmon event (e.g. `searchProcessCreateEvents`), not the GraphQL API sysmonEvents,  Instead of reverting to the existing wording, it would be clearer to change it to "Added “search[sysmon type]Events" Graphql APIs for sysmon event supported by giganto".

Reply:
Modification completed